redis/redisprom/metrics.go

```diff
@@ -0,0 +1,43 @@
+package redisprom
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/reddit/baseplate.go/prometheusbp"
+)
+
+var (
+	// Baseplate.spec Prometheus metrics
+	// https://github.snooguts.net/reddit/baseplate.spec/blob/master/component-apis/prom-metrics.md#redis
+
+	LblPool       = "redis_pool"        // The name of the pool being monitored
```

Comment:
* Don't drop letters to save typing (Label not Lbl)
* These should be constants not variables.
* These should not be in a public package, make it `internal/redisprom`
* 🔕 Consider using a suffix (PoolLabel) not a prefix (LabelPool).
* 🔕 I recommend unit tests for metrics.

Reply:
added all the easy changes, trying to figure out how all this is tested so I can add cases for the metrics